src/components/Delegates/DelegateCardList/DelegateCard.tsx

```diff
@@ -54,11 +54,9 @@ const DelegateCard = ({
               <span className="text-primary font-bold">
                 {formatNumber(delegate.votingPower.total)} {token.symbol}
               </span>
-              {votingStats && (
-                <span className="text-primary font-bold">
-                  {(votingStats?.last_10_props || 0) * 10}% Participation
-                </span>
-              )}
+              <span className="text-primary font-bold">
+                {(votingStats?.last_10_props || 0) * 10}% Participation
+              </span>
```

Comment:
one problem is that while the data is loading it shows "0% participation" -- it doesn't look that bad, I pulled it down and ran it. Might be confusing though